src/blocks/common.blocks/post/_view/post_view_full.bemtree

```diff
@@ -123,7 +123,8 @@ block('post').mod('view', 'full')(
     }),
 
     mode('issue')(function () {
-        var repo = this.ctx.content;
+        var repo = this.ctx.content,
+            isTag = /.*\/\d\.\d\.\d\/.*/.test(this.data.node.url);
```

Comment:
@tavriaforever Can you give a link to an example of such comment in our code? I'll make the same.